pkg/collector/corechecks/system/wincrashdetect/probe/crashparse.go

```diff
@@ -0,0 +1,197 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+//go:build windows
```

Comment:
nit: add empty line to separate from license comment
```suggestion

//go:build windows
```